mteb/models/openai_models.py

```diff
@@ -60,7 +81,12 @@ def _to_numpy(self, embedding_response) -> np.ndarray:
     revision="1",
```

Comment:
Since we changed the implementation we should probably change the revision for all models. (bump it up to "2")

Reply:
We could keep the old version it is nice for reproducibility, but not something that we want to maintain so I wouldn't do it.